app/js/views/reminders.jsx

```diff
@@ -8,46 +8,28 @@ export default class Reminders extends React.Component {
   constructor(props) {
     super(props);
 
+    this.state = {
+      reminders: [],
+    };
+
     this.speechController = props.speechController;
+    this.server = props.server;
 
     moment.locale(navigator.languages || navigator.language || 'en-US');
+  }
 
-    let k = 0;
-
-    const reminders = [
-      {
-        id: k++,
-        recipient: ['Guillaume'],
-        content: 'Get a haircut',
-        datetime: Date.now() + 1 * 60 * 60 * 1000,
-      },
-      {
-        id: k++,
-        recipient: ['Guillaume'],
-        content: 'File a bug',
-        datetime: Date.now() + 2.5 * 60 * 60 * 1000,
-      },
-      {
-        id: k++,
-        recipient: ['Julien'],
-        content: 'Do the laundry',
-        datetime: Date.now() + 1 * 60 * 60 * 24 * 1000,
-      },
-      {
-        id: k++,
-        recipient: ['Sam'],
-        content: 'Attend ping pong competition',
-        datetime: Date.now() + 10.5 * 60 * 60 * 24 * 1000,
-      },
-      {
-        id: k++,
-        recipient: ['Guillaume'],
-        content: 'Attend Swan Lake by the Bolshoi Ballet',
-        datetime: Date.now() + 45 * 60 * 60 * 24 * 1000,
-      },
-    ];
-
-    this.state = { reminders };
+  componentDidMount() {
+    this.server.reminders.getAll()
```

Comment:
Should this work be done here? Because we set state, maybe we should do this work in `componentWillMount`?  I'm a noob wrt React so I'm not sure how these lifecycles work when setting state asynchronously, `componentWillMount` seems more appropriate here though: https://facebook.github.io/react/docs/component-specs.html#mounting-componentwillmount as it would be run on the Server exactly once, and client exactly once.